.travis.yml

```diff
@@ -111,19 +107,24 @@ before_install:
 
 install:
     - |
-      if [[ $DOCKERTAG == "jnlp-slave" ]]
+      if [[ $DOCKERTAGSUFFIX == "jnlp-slave" ]]
       then
         sed "s|@BASE_IMAGE@|condaforge/${DOCKERIMAGE}|" jnlp-slave/Dockerfile.in > $DOCKERIMAGE/Dockerfile
       fi
-    - docker build --build-arg CUDA_VER -t condaforge/$DOCKERIMAGE:$DOCKERTAG -f $DOCKERIMAGE/Dockerfile --no-cache --squash .
+    - export DOCKERTAG=$MINIFORGE_VERSION-$DOCKERTAGSUFFIX
```

Comment:
Oh, note to self: this would produce ugly tags if the suffix is not set. (eg: 4.8.3-0-).